@ui/ui-kit/Textareas/Textarea.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ChangeEvent, FocusEvent } from 'react'
+import classNames from 'classnames'
+import styles from './TextArea.module.scss'
```

Comment:
Your file is called "Textarea.module.scss" but you are importing it as "TextArea.module.scss". This throws an error on vercel